front/lib/services/answer.service.ts

```diff
@@ -24,11 +29,25 @@ class Answer {
     return Answer.instance
   }
 
+  public buildExistingAnswers = (
+    answers: AnswerType[],
+    projectLanguageCode: string
+  ): DefaultAnswerProps[] => {
+    let existingAnswers: DefaultAnswerProps[] = []
+    if (answers) {
+      existingAnswers = answers.map(answer =>
+        AnswerService.buildAnswer(answer, projectLanguageCode)
```

Comment:
tu peux remplacer AnswerService par this ? Ou self ? Je trouve ca chelou de devoir faire ref a AnswerService quand t'es dans AnswerService